documentation/FlowManager.md

```diff
@@ -60,26 +60,67 @@ Note that a separate logout of "reset" is not necessary. The final condition of
 ### User Login (GET /user/:gpiiKey/login)
 
 * **description**: Log in a user to the system
-* **Supported modes**: works only with a locally installed GPII framework (i.e. non-cloud-based flowmanager)
 * **route:** `/user/:gpiiKey/login` where :gpiiKey should be the GPII key of the user
 * **method:** `GET`
 * **return:** Message saying that user successfully logged into the system or an error message.
 
 ### User Logout (GET /user/:gpiiKey/logout)
 
 * **description**: Log out a user of the system
-* **Supported modes**: works only with a locally installed GPII framework (i.e. non-cloud-based flowmanager)
 * **route:** `/user/:gpiiKey/logout` where `:gpiiKey` should be the GPII key of the user
 * **method:** `GET`
 * **return:** Message saying that user successfully logged out of the system or an error message.
 
+## APIs on Cloud Based Flow Manager
+
+### Check the readiness of Cloud Based Flow Manager (GET /ready)
```

Comment:
I think something is backwards here, or maybe the endpoint names are confusing to me. From https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/:

> The kubelet uses liveness probes to know when to restart a Container. For example, liveness probes could catch a deadlock, where an application is running, but unable to make progress. Restarting a Container in such a state can help to make the application more available despite bugs.

> The kubelet uses readiness probes to know when a Container is ready to start accepting traffic. A Pod is considered ready when all of its Containers are ready. One use of this signal is to control which Pods are used as backends for Services. When a Pod is not ready, it is removed from Service load balancers.

This description of `/ready` sounds like a liveness check, while the description of `/health` sounds like a readiness check.

Reply:
Thanks for the clarification on readiness and liveness, @mrtyler. I've swapped their implementations and improved documentation. Ready for another look. Thanks.